Update ZHA sections about Zigbee channel and Adding devices (#27423)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

### Defining Zigbee channel to use

> Tip! Before considering to change to an other Zigbee channel on an existing Zigbee network it is highly recommended that you read through the two segments under the troubleshooting section below about "*Best practices to avoid pairing/connection difficulties*" and "*Zigbee interference avoidance and network range/coverage optimization*" for general prerequisite knowledge and advice on how to achieve the best possible Zigbee network in your environment.

Reviewer: ```suggestion
Tip! Before considering to change to an other Zigbee channel on an existing Zigbee network, it is highly recommended that you read through the two segments under the [troubleshooting](#troubleshooting) section below about "*Best practices to avoid pairing/connection difficulties*" and "*Zigbee interference avoidance and network range/coverage optimization*". These sections provide prerequisite information and advice on how to achieve the best possible Zigbee network in your environment.
```
Rephrased a bit. I think you can provide information but knowledge is something you have to acquire yourself. I think knowledge is in your head, not on a hard drive.

      channels: [15, 20, 25]  # Channel mask
```

> Note! The general best practice is to not change the Zigbee channel from the ZHA default. Also, the related troubleshooting segments mentioned in the tip above will among other things inform that if you have issues with overlapping frequencies between Wi-Fi and Zigbee then it will usually be a better idea to first only try changing and setting a static Wi-Fi channel on your Wi-Fi Router or all your Wi-Fi Access Points (instead of just changing to an other Zigbee channel).

Reviewer: ```suggestion
Note! The best practice is to not change the Zigbee channel from the ZHA default. Also, the related troubleshooting segments mentioned in the tip above will, among other things, inform that if you have issues with overlapping frequencies between Wi-Fi and Zigbee, then it is usually better to first only try changing and setting a static Wi-Fi channel on your Wi-Fi router or all your Wi-Fi access points (instead of just changing to another Zigbee channel).
```

MetaGeek Support has a good reference article about channel selection for [Zigbee and WiFi coexistance]([https://support.metageek.com/hc/en-Ti](https://support.metageek.com/hc/en-us/articles/203845040-ZigBee-and-WiFi-Coexistence)).

> The Zigbee specification standards divide the 2.4Ghz ISM radio band into 16 Zigbee channels (i.e. distinct radio frequencies for Zigbee). For all Zigbee devices to be able to communicate, they must support the same Zigbee channel (i.e. Zigbee radio frequency) that is set on the Zigbee Coordinator as the channel to use for its Zigbee network. Not all Zigbee devices support all Zigbee channels, it will usually depend on the hardware and firmware age as well as devices power ratings.

Reviewer: ```suggestion
The Zigbee specification standards divide the 2.4&nbsp;GHz ISM radio band into 16 Zigbee channels (i.e. distinct radio frequencies for Zigbee). For all Zigbee devices to be able to communicate, they must support the same Zigbee channel (i.e. Zigbee radio frequency) that is set on the Zigbee Coordinator as the channel to use for its Zigbee network. Not all Zigbee devices support all Zigbee channels. Channel support usually depends on the age of the hardware and firmware, as well as on the device's power ratings.
```

> The general recommendation is to only use channels 15, 20, or 25 in order to avoid interoperability problems with Zigbee devices, not only because there is less chance of Wi-Fi networks interfering too much with the Zigbee network on other channels, but also because not all Zigbee devices support all channels. Some devices are for example limited to only being compatible with those ZLL (Zigbee Light Link) channels, it is therefore especially not recommended to use Zigbee channels 11, 24, 25, or 26 on your Zigbee Coordinator as that could mean it will not be usable with older Zigbee devices as those Zigbee channels are commonly only supported by relatively modern Zigbee hardware devices with newer Zigbee firmware.

Reviewer: ```suggestion
The general recommendation is to only use channels 15, 20, or 25 in order to avoid interoperability problems with Zigbee devices. Not only because there is less chance of Wi-Fi networks interfering too much with the Zigbee network on other channels, but also because not all Zigbee devices support all channels. Some devices, for example, are limited to only being compatible with ZLL (Zigbee Light Link) channels. It is therefore especially not recommended to use Zigbee channels 11, 24, 25, or 26 on your Zigbee coordinator. These Zigbee channels are commonly only supported by relatively modern Zigbee hardware devices with newer Zigbee firmware. If using those channels, your coordinator may not be usable with older Zigbee devices.
```

## Adding devices

> Tip! Regardless if your timeline is now before or after you started adding devices to the ZHA integration, it is highly recommended that you read through the two segments under the troubleshooting section below about "*Best practices to avoid pairing/connection difficulties*" and "*Zigbee interference avoidance and network range/coverage optimization*" for general prerequisite knowledge and advice on how to achieve the best possible Zigbee network in your environment.

Reviewer: ```suggestion
Tip! It is highly recommended that you read through the two segments under the troubleshooting section below about "*Best practices to avoid pairing/connection difficulties*" and "*Zigbee interference avoidance and network range/coverage optimization*" for general prerequisite knowledge and advice on how to achieve the best possible Zigbee network in your environment.
```

**To add a new Zigbee device:**

1. Go to the **Integrations** panel, find the **Zigbee Home Automation** integration that was added by the configuration steps above, and select **Configure**.
1. Click on the plus button at the bottom right corner to start a scan for new devices.

> 1. Reset your Zigbee devices to fectory default settings according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.

Reviewer: ```suggestion
1. Reset your Zigbee devices to factory default settings according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times; switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
```

1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.

### Using router devices to add more devices

> You can normally assume that most mains-powered devices, e.g., many always-powered wall plugs or lightbulbs in your Zigbee network will automatically act as a Zigbee Router Device (sometimes also referred to as a Zigbee "signal repeater" or "range extended"). 

Reviewer: ```suggestion
Most mains-powered devices, e.g., many always-powered wall plugs or light bulbs in your Zigbee network will automatically act as a Zigbee router device (sometimes also referred to as a Zigbee "signal repeater" or "range extender"). 
```

Reviewer: Would it make sense to give a bit of high-level information somewhere on the Zigbee device types? Something like: There are three types of Zigbee devices: end devices, routers, and coordinators. Most mains-powered devices, e.g. , many ....

> Because Zigbee should use a "[wireless mesh network]"(https://en.wikipedia.org/wiki/Wireless_mesh_network) to be effective you will need to add Zigbee Router devices to increase the number of Zigbee devices that can be used in your Zigbee network, both in the total number of devices that can be added as well as the total range and coverage of the network. You should also be aware that some Zigbee Router devices do a much better job at routing/repeating Zigbee signals/messages than some other devices, and you should not have a setup where Zigbee Router devices (e.g. lightbulbs) are often powered-off as Zigbee Router devices are meant to be always available.

Reviewer: ```suggestion
Because Zigbee should use a "[wireless mesh network]"(https://en.wikipedia.org/wiki/Wireless_mesh_network) to be effective, you will need to add Zigbee router devices to increase the number of Zigbee devices that can be used in your Zigbee network, both in the total number of devices that can be added as well as the total range and coverage of the network. Some Zigbee router devices do a much better job at routing and repeating Zigbee signals and messages than some other devices. You should not have a setup where Zigbee router devices (e.g. light bulbs) are often powered-off.  Zigbee router devices are meant to be always available.
```

> All Zigbee Coordinator firmware will only allow you to directly connect a certain amount of devices, and that limit is set for two reasons; firstly to not overload the Zigbee Coordinator SoC, and secondly to encourage your Zigbee network to quickly begin to utilize a "[mesh networking](https://en.wikipedia.org/wiki/Mesh_networking)" topology instead of only a "[star network](https://en.wikipedia.org/wiki/Star_network)" topology.

Reviewer: ```suggestion
All Zigbee coordinator firmware will only allow you to directly connect a certain amount of devices. That limit is set for two reasons; firstly, to not overload the Zigbee coordinator, and secondly, to encourage your Zigbee network to quickly begin to utilize a "[mesh networking](https://en.wikipedia.org/wiki/Mesh_networking)" topology instead of only a "[star network](https://en.wikipedia.org/wiki/Star_network)" topology.
```
Removed the system on chip. I think there is already enough technical terminology in there, and I'm not sure that knowing that it's the SoC that is being overloaded makes much of a difference to the user.

> The total number of Zigbee devices that you can have on a Zigbee network depends on a few things, but you should know that the Zigbee Coordinator hardware and its firmware only play a larger role in Zigbee networks with a lot of devices. More important is how many directly connected devices ("direct children") versus how many routers are connected to your Zigbee coordinator. The Zigpy library which the ZHA integration depends on has an upper limit that is 32 direct children, but you can still have hundreds of Zigbee devices in total connected indirectly through routes via Zigbee Router devices.

Reviewer: ```suggestion
The total number of Zigbee devices that you can have on a Zigbee network depends on a few things. The Zigbee coordinator hardware and its firmware only play a larger role in Zigbee networks with a lot of devices. More important is the number of directly connected devices ("direct children") versus the number of routers that are connected to your Zigbee coordinator. The Zigpy library, which the ZHA integration depends on, has an upper limit that is 32 direct children, but you can still have hundreds of Zigbee devices in total connected indirectly through routes via Zigbee router devices.
```

> In this theoretical example, a CC2652 based Zigbee Coordinator has three CC2530 Zigbee Router devices for a total device limit of 77:

Reviewer: ```suggestion
In this theoretical example, a CC2652-based Zigbee coordinator has three CC2530 Zigbee router devices for a total limit of 77 devices:
```

- Coordinator: 32 Zigbee End devices - 3 routers = 29
- Router one: + 16 devices
- Router two: + 16 devices
- Router three: + 16 devices
- Total device limit = **77 devices**

> Note that in practice you will likley need add a lot more Zigbee Router devices than in this example in order to extend the coverage of the network to reach that many devices.

Reviewer: ```suggestion
In practice, you will likely need to add a lot more Zigbee router devices than in this example in order to extend the coverage of the network to reach that many devices.
```
